ansible/roles/atmo-vnc/tasks/realvnc.yml

```diff
@@ -1,7 +1,8 @@
 ---
 
 - name: set environment variables for legacy systems
-  set_fact: ansible_python_interpreter=/usr/bin/python
+  set_fact:
+    ansible_python_interpreter: /usr/bin/python
```

Comment:
I would think once near the top of `tasks/main.yml` will suffice, then it'll run once each time the role is called